debian/build.py

```diff
@@ -40,20 +41,18 @@ def __str__(self):
         self.pkg, self.arch, self.distro, self.version, self.revision)
 
 def main():
-  architectures=('amd64', 'arm', 'arm64')
+  architectures=('amd64',)
```

Comment:
+1, although I'm happy with keeping it as is, if we would rather put effort into something like #89.